/**
 * @author generated by Vogella
 */
package com.vogella.junit.first;

import static org.junit.Assert.assertEquals;
import org.junit.*;

public class MyClassTest {

	//test to check for thrown exception method
	@Test (expected = IllegalArgumentException.class)
	public void testExceptionIsThrown() {
		MyClass tester = new MyClass();
		tester.multiply1(1000,5);
	}
	
	@Test
	public void testMultiply() {
		MyClass tester = new MyClass();
		assertEquals("10 x 5 must be 50", 50, 
				tester.multiply1(10, 5));
	}

}
